azurerm/internal/services/sql/resource_arm_sql_database.go

```diff
@@ -570,6 +580,10 @@ func resourceArmSqlDatabaseRead(d *schema.ResourceData, meta interface{}) error
 		} else {
 			d.Set("read_scale", false)
 		}
+
+		if zoneRedundant := props.ZoneRedundant; zoneRedundant != nil {
+			d.Set("zone_redundant", zoneRedundant)
+		}
```

Comment:
since this is a simple type we can update this:

```suggestion
		d.Set("zone_redundant", props.ZoneRedundant)
```